pkg/collector/corechecks/ebpf/c/runtime/oom-kill-kern-user.h

```diff
@@ -19,12 +18,14 @@ struct oom_stats {
     char trigger_comm[TASK_COMM_LEN];
     // OOM score of killed process
     __s64 score;
-    // OOM score adjustment of killed process
-    __s16 score_adj;
     // Total number of pages
     __u64 pages;
     // Tracks if the OOM kill was triggered by a cgroup
     __u32 memcg_oom;
+    // OOM score adjustment of killed process
+    __s16 score_adj;
+
+    char cgroup_name[129];
```

Comment:
is this a required layout in context of this PR? add comment in code if that's the case

Reply:
all of the struct changes are. Do you want comments for each?